xarray/core/duck_array_ops.py

```diff
@@ -101,6 +101,9 @@ def isnull(data):
 einsum = _dask_or_eager_func('einsum', array_args=slice(1, None),
                              requires_dask='0.17.3')
 
+masked_invalid = _dask_or_eager_func('masked_invalid', eager_module=np.ma,
+                                     dask_module=dask_array.ma)
```

Comment:
Unfortunately this breaks if dask is not installed:
```
xarray/__init__.py:6: in <module>
    from .core.alignment import align, broadcast, broadcast_arrays
xarray/core/alignment.py:11: in <module>
    from .indexing import get_indexer_nd
xarray/core/indexing.py:11: in <module>
    from . import duck_array_ops, nputils, utils
xarray/core/duck_array_ops.py:105: in <module>
    dask_module=dask_array.ma)
E   AttributeError: 'NoneType' object has no attribute 'ma'
```

Instead, we need something like `dask_module=getattr(dask_array, 'ma', None)`